### DISCUSSION TITLE
Error during deployment

### DISCUSSION THREAD

**Discussion by Piyushwebdev** [Atri-Labs/atrilabs-engine#632] [Help: Miscellaneous] [answered]
How to deploy atri app on netlify ? While trying to deploy it on github i am getting errors like "Assser_URL_Prefix command not found" Similar error for other commands. Please help me on it, I need to submit this project after deploying.

**Comment by darshitac11** [comment, marked as answer]
To deploy to netlify, you will have to push the `atri_app/dist/ssg` folder to netlify.

`atri deploy ssg` doesn't work for Windows users yet. We are working on it.

**Reply by Piyushwebdev** [reply]
but inside atri_app/dist/ there is no folder named as ssg instead it has three folders naming ( app,app-node,server) . I tried to deploy app inside netlify but images are not visible in the deployed app.
